src/cryptonote_core/blockchain.cpp

```diff
@@ -441,6 +441,55 @@ bool Blockchain::init(BlockchainDB* db, const network_type nettype, bool offline
   MINFO("Blockchain initialized. last block: " << m_db->height() - 1 << ", " << epee::misc_utils::get_time_interval_string(timestamp_diff) << " time ago, current difficulty: " << get_difficulty_for_next_block());
   m_db->block_txn_stop();
 
+  uint64_t num_popped_blocks = 0;
+  while (true)
+  {
+    uint64_t top_height;
+    crypto::hash top_id;
+    top_id = get_tail_id(top_height);
+    bool orphan;
+    block top_block;
+    get_block_by_hash(top_id, top_block, &orphan);
+    const uint8_t ideal_hf_version = get_ideal_hard_fork_version(top_height);
+    if (ideal_hf_version <= 1 || ideal_hf_version == top_block.major_version)
+    {
+      if (num_popped_blocks > 0)
+        MGINFO("Initial popping done, top block: " << top_id << ", top height: " << top_height << ", block version: " << (uint64_t)top_block.major_version);
```

Comment:
MGINFO is a shortcut for global category, you want MINFO

Reply:
OK